operator/api/v1alpha1/securedcluster_types.go

```diff
@@ -104,11 +104,22 @@ type SecuredClusterSpec struct {
 	//+operator-sdk:csv:customresourcedefinitions:type=spec,displayName="Custom Default Image Registry",order=14,xDescriptors={"urn:alm:descriptor:com.tectonic.ui:advanced"}
 	RegistryOverride string `json:"registryOverride,omitempty"`
 
+	// Controls if process baselines are locked once the observation period ends
+	//+operator-sdk:csv:customresourcedefinitions:type=spec,order=3
+	AutoLockProcessBaselines *AutoLockProcessBaselinesSpec `json:"autoLockProcessBaselines,omitempty"`
+
 	// Network configuration.
 	//+operator-sdk:csv:customresourcedefinitions:type=spec,displayName=Network,order=15,xDescriptors={"urn:alm:descriptor:com.tectonic.ui:advanced"}
 	Network *GlobalNetworkSpec `json:"network,omitempty"`
 }
 
+// Controls if process baselines are locked once the observation period ends
+type AutoLockProcessBaselinesSpec struct {
+	// When set to true process baselines are locked when the observation period ends
+	//+operator-sdk:csv:customresourcedefinitions:type=spec,order=3
+	Enabled *bool `json:"enabled,omitempty"`
```

Comment:
```suggestion
	// Should process baselines be automatically locked when the observation period (XX minutes by default) ends.
	// The default is: Auto.
	//+operator-sdk:csv:customresourcedefinitions:type=spec
	Lock *ProcessBaselineLockMode `json:"lock,omitempty"`
```

and above you'll need to define `type ProcessBaselineLockMode string` as `Enum=Auto;Disabled` with a couple of const definitions.